Podfile

```diff
@@ -179,7 +186,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.85.1'
+  gutenberg commit: 'ed6de0ebcdfd1b6d2899bcdf045ccc307e923ea9'
```

Comment:
Based on previous changes, seems that this line should invoke the `gutenberg_pods` function. I checked the recent commits and [this one](https://github.com/wordpress-mobile/WordPress-iOS/pull/19326/commits/19c1285d0392f43796c4481b23e94c1ed2622231) related to updating the branch with `trunk` is likely the one that introduced it. I'm wondering if it was caused by conflict resolution or if something went wrong when auto-merging 🤔 . In any case, we should revert the change and point to the new `gutenberg_pods`.

Reply:
Good catch. I've updated this to match the format from the `trunk` Podfile.